hudi-common/pom.xml

```diff
@@ -221,14 +221,13 @@
       <groupId>org.apache.hbase</groupId>
       <artifactId>hbase-client</artifactId>
       <version>${hbase.version}</version>
-      <scope>test</scope>
```

Comment:
this may be ok per se, given the hbase server is 2.x. it may no longer work against a HBase 1.x server and we should call this out in the release notes